src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/ApplicationInsightsTelemetryConverter.cs

```diff
@@ -48,32 +40,22 @@ public override IEnumerable<ITelemetry> Convert(LogEvent logEvent, IFormatProvid
                 return _exceptionTelemetryConverter.Convert(logEvent, formatProvider);
             }
 
-            if (logEvent.MessageTemplate.Text.StartsWith(MessagePrefixes.RequestViaHttp))
+            if (logEvent.Properties.ContainsKey(ContextProperties.RequestTracking.RequestLogEntry))
```

Comment:
We should rely on the new approach: https://github.com/arcus-azure/arcus.observability/pull/215/files#diff-0defdc0c79223504c82f4ce48ee49faccfb8fdf16640651e1747420d5ded944eR52

Reply:
But the context is part of the logentry type.. so we do have to use the message format to retrieve this entry, right? 
And since we already know by the kind of log entry that has been written, why would we have to deserialize the log entry into a Serology value, retrieve the context telemetry from that value, retrieve the telemetry type value from that context to get something we already know?

Reply:
I'm not following anymore.  I think a short offline discussion can remove the clouds :)